eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -1,30 +1,32 @@
 {
   "displayNames": {
-    "/p:UseProjectReferenceToAzureClients=true": "UseProjectRef",
-    "true": "Record"
+    "/p:UseProjectReferenceToAzureClients=false": "PackageRef",
+    "/p:UseProjectReferenceToAzureClients=true": "ProjectRef"
   },
   "matrix": {
     "Agent": {
       "ubuntu-18.04": { "OSVmImage": "MMSUbuntu18.04", "Pool": "azsdk-pool-mms-ubuntu-1804-general" },
       "windows-2019": { "OSVmImage": "MMS2019", "Pool": "azsdk-pool-mms-win-2019-general" },
       "macOS-10.15": { "OSVmImage": "macOS-10.15", "Pool": "Azure Pipelines" }
     },
-    "TestTargetFramework": [ "netcoreapp2.1", "net5.0" ]
+    "TestTargetFramework": [ "netcoreapp2.1", "net461", "net5.0" ],
+    "AdditionalTestArguments": [ "/p:UseProjectReferenceToAzureClients=false", "/p:UseProjectReferenceToAzureClients=true" ]
```

Comment:
For longer elements like this at least, can we put these array elements on separate lines? Easier to read, cleaner diffs.